src/OSPSuite.Infrastructure.Import/Services/IDataImporter.cs

```diff
@@ -1,10 +1,24 @@
 ﻿using OSPSuite.Core.Domain.Data;
 using OSPSuite.Infrastructure.Import.Core;
 using System.Collections.Generic;
+using System.Linq;
 using ImporterConfiguration = OSPSuite.Core.Import.ImporterConfiguration;
 
 namespace OSPSuite.Infrastructure.Import.Services
 {
+   public class ReloadDataSets
+   {
+      public IEnumerable<DataRepository> NewDataSets;
```

Comment:
{get;set} and in fact probably only get and set in constructor.
Always think immutable if possible

Reply:
fixed